internal/metadata/meta_test.go

```diff
@@ -16,8 +16,11 @@ func TestParseQueryNameAndType(t *testing.T) {
 		`--name: CreateFoo :two`,
```

Comment:
I could write a much nicer implementation of the parse function (using a word scanner) if we treat `--name: ...` as either (1) valid name and cmd metadata or (2) not invalid, but just ignored and treated like a regular comment.

I will also point out that we don't special case the treatment of a missing space between comment start and `name:` with the other two supported comment syntaxes.